Kconfig

```diff
@@ -28,6 +35,17 @@ config SCCB_HARDWARE_I2C
         Enable this option if you want to use hardware I2C to control the camera.
         Disable this option to use software I2C.
 
+config ENABLE_TEST_PATTERN
```

Comment:
I really do not see the point of having this setting here, given that you can just get the sensor and call the function in your code. This is a driver and not a final application. Please remove it.